pkg/reconciler/vertex/controller.go

```diff
@@ -160,6 +160,56 @@ func (r *vertexReconciler) reconcile(ctx context.Context, vertex *dfv1.Vertex) (
 		return ctrl.Result{}, err
 	}
 
+	// Create services
```

Comment:
Can you please add a comment on why this "move" is important? A summary of the PR description is good enough.

Any constants should be applied before the runtime variables can come into play.